### ISSUE TITLE
Data Collection Collector sentence

### ISSUE THREAD

**Issue by okilker** [open-telemetry/opentelemetry.io#491]
The Collector is the default location instrumentation libraries export their telemetry data.

⬆️ This sentence seems like it could be unpacked a little. (for greenfield environments), or perhaps a word or two could be added for clarification?:

- The Collector is the default location where instrumentation libraries export their telemetry data.
OR
- The Collector is the locally-running default location where instrumentation libraries export their telemetry data.
OR
- The Collector (source url? API?) is the default location where (local?) instrumentation libraries such as, X export their telemetry data to to the user’s desired back-end(s)

Later, here in the reference architecture paragraph it says: https://opentelemetry.io/docs/

"By default, these instrumentation libraries are configured to export their data to a **locally running** Collector."

OpenTelemetry Collector should be deployed:
- as an agent on each host within an environment
- configured to send telemetry data to the user’s desired back-end(s). 
- OpenTelemetry instrumentation libraries need to be added to each application. 
These instrumentation libraries are configured by default, to export their data to a locally running Collector. 

Optionally, a pool of Collector instances can be deployed in a region. (how to do this link?)

**Comment by johnmezzetta** [comment]
I agree that the sentence is really vague. The java example does not show the creation/configuration of a collector in conjunction with the java instrumentation. So it would be reasonable to assume that the collector logic is imbedded in java instrumentation. However, that might be a bad assumption. Can a clear statement be made as to whether a given instrumentation has an imbedded collector? For example, the java example for Prometheus exposes the localhost:19090/metrics endpoint for scraping. Is this a function of an imbedded collector that serves up this endpoint or does the sentence mean that the default configuration of a locally installed collector is set to scrape localhost:19090/metrics for input metrics data?

**Comment by cartermp** [comment]
The collector page was updated a little while ago to address this topic: https://opentelemetry.io/docs/collector/#when-to-use-a-collector

Additionally, we've started to normalize Getting Started and Exporters pages for all the languages to show how to send to a local collector, but also mention that it's the same as general OTLP export. It's not 100% complete, but it's an established pattern that will likely be filled in for most languages in time.

Given this, I'll probably close this issue.
